packages/core/documentation/Link/LinkExample.tsx

```diff
@@ -4,15 +4,15 @@ import { ExampleComponentProps } from "../../../../doc-utils";
 import { Link } from "../../src";
 
 export const LinkExample: VFC<ExampleComponentProps> = ({ boolValues }) => (
-    <p className="jkl-body">
+    <p className={boolValues?.["Compact"] ? "jkl-small" : "jkl-body"}>
```

Comment:
Litt usikker på om jeg synes dette gir mening. Her gir vi jo inntrykk av at det finnes en "compact" prop på lenker, men de arver jo tekststilen som er brukt. Går ut fra at dette kommer fra Figma-variantene, men kanskje det er bedre å synkronisere andre veien?

Foreslår følgende:
* Endre til en `choiceProp` som heter "Tekststil" (eller tilsvarende på engelsk) med "body" og "small" (og kanskje "lead"?) som valg, og la den styre klassen på avsnittet
* Lage tilsvarende varianter i Figma og endre navnet på variablen der til "Tekststil" (eller engelsk versjon)

Reply:
@Slaine2 tagger deg også her siden jeg ser at du etterspurte endringen 😊

Reply:
Figma-variantene var utgangspunktet før ønsket fra @Slaine2 ja.

Jeg ser hvor du kommer fra. Jeg tenker vi dekker oss inn godt nok ved å ha kodeeksempelet lett tilgjengelig, og at det fort blir tydelig at det ikke finnes noen `forceCompact`-prop ved at TypeScript klager.

Begge deler har verdi da:

1. Behandle Compact som "bare gjør den mindre, jeg bryr meg ikke om detaljene"
2. "Compact betyr at det er en forceCompact prop"

Det første forenkler mappingen mellom Figma og portalen, og "lekker" ikke implementasjonsdetaljer fra kode helt inn i Figma-biblioteket. Det siste forenkler mappingen mellom portalen og React.

Jeg heller mot at utviklerne har "den minste ulempe" i akkurat denne situasjonen, med løsningen slik den er nå.

Reply:
Enig i at det er utviklerne som har "minste ulempe" slik det er nå ja 😊 Tenker bare vi skal være forsiktige med begrepsbruk, siden ting har en tendens til å feste seg. Særlig siden de andre endringene i PRen understøtter en 1-til-1 sammenheng mellom egenskaper i kode, Figma og eksempel.

Jeg tror vi kan ha det som det er og det vil funke greit, men i dette tilfellet tenker jeg at det er snakk om litt mer enn en implementasjonsdetalj: Det finnes jo egentlig ikke noe konsept som heter "kompakt lenke" i designsystemet (slik jeg forstår det), så i mine øyne er det feil å fremstille det slik _både_ i Figma og i eksempelet (derav forslaget over).

Hvis det derimot _er_ slik at vi vil ha kun to varianter av lenkene synes jeg at vi faktisk bør implementere en prop som setter riktig tekststil på komponenten. Dette er ikke blocking for denne PRen altså, men jeg synes det er greit å ta en diskusjon på alle mismatcher på konseptnivå når de dukker opp 😊